openedx/core/djangoapps/user_authn/cookies.py

```diff
@@ -153,29 +157,14 @@ def refresh_jwt_cookies(request, response):
             refresh_token = request.COOKIES[jwt_cookies.jwt_refresh_cookie_name()]
         except KeyError:
             raise AuthFailedError(u"JWT Refresh Cookie not found in request.")
-        _create_and_set_jwt_cookies(response, request, refresh_token=refresh_token)
-    return response
-
-
-def _set_deprecated_logged_in_cookie(response, request):
-    """ Sets the logged in cookie on the response. """
-
-    # Backwards compatibility: set the cookie indicating that the user
-    # is logged in.  This is just a boolean value, so it's not very useful.
-    # In the future, we should be able to replace this with the "user info"
-    # cookie set below.
-    cookie_settings = standard_cookie_settings(request)
-
-    response.set_cookie(
-        settings.EDXMKTG_LOGGED_IN_COOKIE_NAME.encode('utf-8'),
-        'true',
-        **cookie_settings
-    )
 
+        # TODO don't extend the cookie expiration - reuse value from existing cookie
+        cookie_settings = standard_cookie_settings(request)
```

Comment:
Couldn't this be moved into `_create_and_set_jwt_cookies` using the following line, like you added to `set_deprecated_user_info_cookie`?
```
cookie_settings = cookie_settings or standard_cookie_settings(request)
```

Reply:
The `set_deprecated_user_info_cookie` is a public function and is called with and without the `cookie_settings`.  Both variations are therefore covered in our unit tests.  Since `_create_and_set_jwt_cookies` is an internal method and only called one-way, we don't need this generality.